### DISCUSSION TITLE
scrypt support?

### DISCUSSION THREAD

**Discussion by Xynonners** [frankie567/pwdlib#11] [Ideas]
Hi,

hashlib in the standard library seems to have an implementation of pbkdf2 and scrypt. Maybe we can link up to those?

**Comment by frankie567** [comment]
Yes, `scrypt` support could be really interesting! I had actually a look when I started the lib; the main difficulty is to come up with a dedicated format for the resulting hash that embeds the parameters, salt and the actual hash. Ideally, one that would be compatible with `passlib`.
